README.md

```diff
@@ -15,9 +15,11 @@ For use of the raw data itself, all can be found in `output-data` directory, inc
 3. `pip install poetry`
 4. `python -m poetry install`
 5. `python -m pre_commit install`
-6. Download Decompiler.exe for your particular OS from https://github.com/ValveResourceFormat/ValveResourceFormat/releases 
-7. Extract decompiler into `Downloads/` folder
-8. Ensure the paths to Deadlock files and Decompiler are correct in `src/parser/decompile.sh`
+6. `pip install keyvalues3`
+7. `pip install python-mermaid`
```

Comment:
Not sure why these deps are not behaving. Ideally poetry should handle all of the dependencies